lib/containers/utils.pm

```diff
@@ -41,52 +41,69 @@ sub test_seccomp {
     }
 }
 
+sub setup_registry {
+    my $runtime  = shift;
+    my $registry = get_var('REGISTRY');
+    if ($registry) {
+        assert_script_run('openssl s_client -servername ' . $registry . ' -connect ' . $registry . ':443 < /dev/null 2>/dev/null | openssl x509 > /etc/pki/trust/anchors/certificate.crt');
```

Comment:
If the registry is internal, then the system should already have the right certificate (also needed for `registry.suse.de`) and if it's external, it should have a valid certificate anyway

Reply:
Actually, there's `allow_selected_insecure_registries` already. Could that be merged/coordinated with `allow_selected_insecure_registries`?

Reply:
It's external. We need a CA authority to make it valid and that costs money. That's why I'm doing this. The other option is making the registry insecure and we don't care about certificates.

Reply:
> It's external. We need a CA authority to make it valid and that costs money

https://letsencrypt.org/?

Reply:
After some research, I think I could just set the registry as insecure and re-use the function [allow_selected_insecure_registries](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/7d811c1afc2985cd208ebe2907175b28f24b0ace/lib/containers/common.pm#L98)

This way we avoid issuing new certificates if we need to move around the `REGISTRY` instance.